flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/OneInputOperatorWrapper.java

```diff
@@ -52,7 +53,9 @@ public OneInputOperatorWrapper(GeneratedClass<OneInputStreamOperator<IN, OUT>> g
 	public void setup(StreamTask<?, ?> containingTask, StreamConfig config,
 			Output<StreamRecord<OUT>> output) {
 		operator = generatedClass.newInstance(containingTask.getUserCodeClassLoader());
-		operator.setup(containingTask, config, output);
+		if (operator instanceof SetupableStreamOperator) {
+			((SetupableStreamOperator) operator).setup(containingTask, config, output);
```

Comment:
Would it be viable to migrate this already to the non setupable operator? So to replace this if check with:
```
operator = generatedClass.newInstance(containingTask.getUserCodeClassLoader(), containingTask, config, output);
```
?

If it's non trivial it could be done as some kind of follow up step.